content/en/news/2019-03-25-server-outage-aftermath.md

```diff
@@ -0,0 +1,139 @@
+maintenance overhead. As the versions in use were different, it was
+not possible to just copy the database over from the backup of
+2018-10-28. This has the following consequences:
+
+* If you registered to the web interface or subscribed to the
+  tsc-devel mailing list after 2018-10-28, you lost your account and
+  subscription. You need to register and/or subscribe again.
+* If you were subscribed before 2018-10-28 and use only e-mail to interact
+  with the tsc-devel mailing list, you don't need to do anything. You
+  have already been subscribed again.
+* If you were registered before 2018-10-28 and use the [web forum
+  interface][7] to interact with tsc-devel, you have been switched to
+  an e-mail-only subscription for technical reasons, i.e. the forum
+  account was lost. You need to go to the [web forum interface][7]
+  and re-register with the address you receive e-mails from the forum
+  at. The forum will automatically match your subscribed e-mail
+  address to the forum account you create. Once you have logged in,
+  you can configure e-mail notifications of posts by clicking on
+  “Manage Lists” on the top right (alternatively, [follow this link
+  after login][8]). Click the red “Unsubscribe” button on the page
+  for the tsc-devel mailing list to stop receiving e-mails from the
+  list. You can still post via the web interface after doing this.
+* If you want to delete your subscription without creating a forum
+  account, please join our IRC channel `#secretchronicles` on
+  chat.freenode.net and request this there.
+
+The other mailing list, tsc-users, has not been restored. It never saw
+any serious traffic anyway. All discussion, even if it is user
+discussion like submission of levels, should just happen on tsc-devel
+from now on. All subscriptions to the tsc-users mailing list have been
+cancelled.
+
+We apolgise for the inconvenience caused by the Mailman version switch.
+
+## Future
+
+We are going to evaluate our backup strategy and quickly get automatic
+backups up and running for the new server.
+
+Additionally, beyond restoring the services that we had on the old
+VPS, we consider setup of more infrastructure. For instance, the Wiki,
+which we set to sleep after it got spammed might see a
+rennaissance. The setup of a CI service is in discussion, as are
+automated nightly builds. TSC team member kirbyfan suggested
+installation of Discourse, a rails-based web forum software.
+
+quintus<br/>
+on behalf of the TSC team.
+
+[1]: https://github.com/Secretchronicles/TSC/issues/649
+[2]: https://www.first-root.com/
+[3]: https://github.com/Secretchronicles/TSC/issues/649
+[4]: https://chatlogs.secretchronicles.org
+[5]: https://wekan.github.io/
+[6]: https://lists.secretchronicles.org/hyperkitty
+[7]: https://lists.secretchronicles.org/hyperkitty/list/tsc-devel@lists.secretchronicles.org/
+[8]: https://lists.secretchronicles.org/postorius/lists/
+[9]: https://github.com/Secretchronicles/TSC
```

Comment:
Add:
[10]: https://blog.wekan.team/2018/01/wekan-progress-on-x64-and-arm/index.html